### ISSUE TITLE
tap2rpm's default long description should mention tap2rpm, not tap2deb

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4083]
|<img alt="TimAllen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| TimAllen reported|
|-|-|
|Trac ID|trac#4083|
|Type|enhancement|
|Created|2009-11-12 05:07:36Z|

Currently, twisted.scripts.tap2rpm contains the following line of code:
```
long_description = (config['long_description']
		    or 'Automatically created by tap2deb')
```

It should of course say "Automatically created by tap2rpm".

Attachments:

* [4083.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/916/916163bf91f3934b1cec3b452e310195b598ec94/c49f05897596ee3c2334ad11e150f1e85dcd7b43.patch) (2627 bytes) - added by Poincare on 2009-11-21 04:03:56Z - reivew
* [4083.2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/916/916163bf91f3934b1cec3b452e310195b598ec94/38fbbc1bf123b754a5fd1b99d14f10b6cdbf8e90.patch) (2627 bytes) - added by Poincare on 2009-11-21 04:07:15Z - Please reivew

<details><summary>Searchable metadata</summary>

```
trac-id__4083 4083
type__enhancement enhancement
reporter__TimAllen TimAllen
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1258002456000000 1258002456000000
changetime__1259698030000000 1259698030000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="TimAllen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|TimAllen commented|
|-|-|

This ticket split off from [#3292](https://github.com/twisted/twisted/issues/3292).

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)|@jonathanj removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)<a name="note_3"></a>|@jonathanj set owner to Poincare|
|-|-|

Perhaps I'm confused, but this ticket talks about tap2rpm's description while the patch also deletes `aimbot.py`. What's up with that?

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

I'll create a new patch and apply it.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra commented|
|-|-|

New patch:

```
#!diff
Index: twisted/scripts/tap2rpm.py
===================================================================
--- twisted/scripts/tap2rpm.py	(revision 27633)
+++ twisted/scripts/tap2rpm.py	(working copy)
@@ -225,7 +225,7 @@
     description = config['description'] or ('A TCP server for %(protocol)s' %
                                             vars())
     long_description = (config['long_description']
-                        or 'Automatically created by tap2deb')
+                        or "Automatically created by tap2rpm")
     twistd_option = type_dict[config['type']]
     date = time.strftime('%a %b %d %Y', time.localtime(time.time()))
     directory = rpm_file + '-' + version
```

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [27636]) Fix the default long description of tap2rpm, that incorrectly
was talking about tap2deb.

Author: Poincare
Reviewer: jonathanj, thijs
Fixes: [#4083](https://github.com/twisted/twisted/issues/4083)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
